tests/test_explorer.py

```diff
@@ -0,0 +1,172 @@
+"""Unit tests for explorer module."""
+
+import unittest
+from unittest import mock
+
+from PyQt5.QtCore import QObject, Qt
+from PyQt5.QtWidgets import QApplication, QTreeWidgetItem
+from PyQt5.QtTest import QTest
+
+from iquip.apps import explorer
+
+class ExplorerFrameTest(unittest.TestCase):
+    """Unit tests for ExplorerFrame class."""
+
+    def setUp(self):
+        self.qapp = QApplication([])
+        patcher = mock.patch("iquip.apps.explorer._FileFinderThread")
+        self.MockedFileFinderThread = patcher.start()
+        self.addCleanup(patcher.stop)
+
+    def tearDown(self):
+        del self.qapp
+
+    @mock.patch("iquip.apps.explorer.ExplorerApp.lazyLoadFile")
+    def test_file_tree_item_expanded(self, mockedLazyLoadFile):
+        app = explorer.ExplorerApp(name="name", masterPath="masterPath", parent=QObject())
+        directoryItem = QTreeWidgetItem(app.explorerFrame.fileTree)
+        directoryItem.setText(0, "directory")
+        QTreeWidgetItem(directoryItem)  # Add an empty item to an unloaded directory.
+        directoryItem.setExpanded(True)
+        mockedLazyLoadFile.assert_called_once()  # pylint: disable=no-member
```

Comment:
Now `mockedLazyLoadFile` is a mock object, so you can remove the pylint comment.

Reply:
Oh, thanks for notifying it! But the tests will be removed..😅